include/stl2/detail/algorithm/max.hpp

```diff
@@ -23,19 +23,17 @@
 // max [alg.min.max]
 //
 STL2_OPEN_NAMESPACE {
-	namespace __max {
-		template<InputRange Rng, class Comp, class Proj>
-		requires
-			Copyable<iter_value_t<iterator_t<Rng>>> &&
-			IndirectStrictWeakOrder<
-				Comp, projected<iterator_t<Rng>, Proj>>
-		constexpr iter_value_t<iterator_t<Rng>>
-		impl(Rng&& rng, Comp comp, Proj proj)
+	class __max_fn {
```

Comment:
`struct` instead of `class`, please: cmcstl2 uses `struct` for class definitions, `class` for template parameters, and `typename` for disambiguation. Each keyword has one unambiguous use which eases grepping.